packages/framework/src/Resources/views/Admin/Content/Article/list.html.twig

```diff
@@ -1,7 +1,13 @@
 {% extends '@ShopsysFramework/Admin/Layout/layoutWithPanel.html.twig' %}
 
 {% block title %}- {{ 'Articles overview'|trans }}{% endblock %}
-{% block h1 %}{{ 'Articles overview'|trans }} <br> ?{% endblock %}
+{% block h1 %}{{ 'Articles overview'|trans }}
```

Comment:
maybe it would be better to put the question mark at the top of `{% block main_content %}` - because now, you have `<h6>` tag inside `<h1>` tag which feels really weird :slightly_smiling_face: